fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/HomeScreenRobot.kt

```diff
@@ -723,13 +723,10 @@ class HomeScreenRobot {
             return TabDrawerRobot.Transition()
         }
 
-        fun expandCollection(title: String, rule: ComposeTestRule, interact: CollectionRobot.() -> Unit): CollectionRobot.Transition {
-            homeScreenList().waitForExists(waitingTime)
-            homeScreenList().scrollToEnd(LISTS_MAXSWIPES)
-
-            collectionTitle(title, rule)
-                .assertIsDisplayed()
-                .performClick()
+        fun expandCollection(title: String, interact: CollectionRobot.() -> Unit): CollectionRobot.Transition {
+            assertItemContainingTextExists(itemContainingText(title))
+            itemContainingText(title).clickAndWaitForNewWindow(waitingTimeShort)
```

Comment:
I'm not sure if clickandWaitForNewWindow is the appropriate solution, it can't hurt because it doesn't return anything to block us. But out of caution, since we have the waitForExists on the next item, that could be enough. If this helps, feel free to keep it.